### PR TITLE
DQN example huber loss and typo, issue #82

### PR DESCRIPTION
**Pull request by dcharrezt**
Issue #82 
Adding baselines package need to run the notebook,
Correcting small typo
Changing huber loss function for tf2

### CODE REVIEW

File: examples/rl/deep_q_network_breakout.py [hunk 1]
```diff
@@ -209,7 +209,7 @@
                 q_action = tf.reduce_sum(tf.multiply(q_values, masks), axis=1)
                 # Calculate loss between new Q-value and old Q-value
                 # Clip the deltas using huber loss for stability
-                loss = keras.losses.huber(updated_q_values, q_action)
+                loss = tf.compat.v1.losses.huber_loss(updated_q_values, q_action)
```
Comment (fchollet) [inline examples/rl/deep_q_network_breakout.py:212]: This should be `tf.reduce_sum(keras.losses.huber(...))`, see https://keras.io/api/losses/regression_losses/#huber-function

Or alternatively 


```
loss_fn = Huber()  # Outside the loop
loss = loss_fn(...)
```

Comment (dcharrezt) [reply examples/rl/deep_q_network_breakout.py]: Thanks for the correction, since `keras.losses.huber` was added in tf-nightly, I am adding the command to install it in the colab.

Comment (fchollet) [reply examples/rl/deep_q_network_breakout.py]: Please remove the commands, the file already explicitly mentions that you need to install tf-nightly.

Comment (dcharrezt) [reply examples/rl/deep_q_network_breakout.py]: Those commands were removed! thanks for pointing that out.
